Formula/elasticsearch.rb

```diff
@@ -12,6 +13,10 @@ class Elasticsearch < Formula
     sha256 "20056758a9acc1f9de8d4adac2fde028154a1b5b1ce4d7062474a69bc1595233" => :mojave
   end
 
+  # elasticsearch will be relicensed before v7.11.
+  # https://www.elastic.co/blog/licensing-change
+  deprecate! date: "2020-01-14", because: "is switching to an incompatible license"
```

Comment:
@Rylan12 I think there's still a way to tweak the message that's actually displayed to users to make both that message and what shows up in the formula as less awkward to read... It'll be on my to-do list.

Reply:
Awesome, I'd love to see that improved!